pkg/pgmodel/metrics/query.go

```diff
@@ -21,7 +21,7 @@ var (
 			Subsystem: "query",
 			Name:      "requests_total",
 			Help:      "Number of query requests to Promscale.",
-		}, []string{"type", "handler", "code"},
+		}, []string{"type", "handler", "code", "err"},
```

Comment:
IMHO this should be called `reason` and not `err` as the latter is not descriptive at all.

Reply:
Totally agree @niksajakovljevic do you mind if I open a new PR and change it to reason before doing the new release? `reason` is more in line with what other tools use.

Reply:
Sure go ahead